### DISCUSSION TITLE
How to use Seleniumbase as a replacement for UC ?

### DISCUSSION THREAD

**Discussion by MacMarde** [ultrafunkamsterdam/undetected-chromedriver#1525] [Q&A] [answered]
It seems from what I can read in the Issues that UC it is not bypassing CF anymore for weeks.
I also read that Seleniumbase might still work (for some reason) and is also using UC.
So I started reading the docs and the Issue threads for both projects on Github.

It seems like there are several approaches how to use Seleniumbase with UC=true and I am not sure which might be the best for my needs or if it will work at all.
The reason is I do have a larger project realized with UC.
Now my goal is to be undetectable again using Seleniumbase with UC=true with as less code changes as possible.

What I am doing now with UC and want to achieve with Seleniumbase is:
- Using multithreading to run several chrome instances, each instance separated from the others and each instance using its own profile-folder.
- Bypassing CF captcha (nowsecure.nl) and be undetectable in every situation.
- Using chrome_options, for example chrome_options.add_argument() but also change the profile, which only seems to be possible using baseCase and overwrite the get_new_driver function. See [here](https://github.com/seleniumbase/SeleniumBase/discussions/1221).

I am not sure if it will be possible and which is the best approach to realize it with less code changes.

**Comment by mshot** [comment, marked as answer]
i sofar ended with something like: 
```
driver = webdriver(browser='chrome', undetectable = True, uc_subprocess=True, proxy=proxy, chromium_arg=f'--remote-debugging-port={port}', user_data_dir=profile)
```
but i'm not really sure about "_be undetectable in every situation_" - with or without custom profile it wasn't 100% (also posible i'm doing something wrong

**Comment by MacMarde** [comment]
@mshot
Thanks. I will try something similar and see if it will work.

**Reply by MacMarde** [reply]
@mshot 

Can you bypass CF captcha here: [www.nowsecure.nl](www.nowsecure.nl) when using user_data_dir ?
It seems not to work for me.
